archetype/api/controllers/ViewController.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+module.exports = {
+
+  helloWorld (request, reply) {
+    reply('Hello Trails.js !')
```

Comment:
I think unless a view is being rendered, we always want to reply in JSON format.